hook-tests/031-faillock.test

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
(nitick) - this could be `/bin/sh` as well, right? Nothing bash specific in here?

Reply:
Why do we want `/bin/sh`?

Reply:
There is no deep reason, it's fine (hence the "nitpick"). It's just that on Ubuntu /bin/sh is dash which is slightly faster than bash but that also does not really matter. I guess it's a bit of a habit because Debian maintainer scripts are encouraged to run with /bin/sh (because there the speed actually adds up over thousands of sh runs from the maintainer scripts) and because in theory one could have a system without bash - but this is not really relevant here and as you can see all of my arguments are rather weak :) So fine to leave if you prefer that.

Reply:
This is a very good explanation. Thank you.